.envdefault

```diff
@@ -46,4 +46,5 @@ NO_REPLY_EMAIL=
  
 VOLUME_FOLDER=storages
 VOLUME_AUDIO_PATH=audios
+VOLUME_AUDIO_WAVEFORM_PATH=audiowaveform
```

Comment:
You can  VOLUME_XXXX on the .envdefault
Those should not exist, i still need to make some clean on my part. We don't need for the user to configure the folder location. On docker it will just be a volume.

You will need to : 
 - Add in config.js (process.env.VOLUME_AUDIO_WAVEFORM_PATH = 'audiowaveform'
 - Add in docker-entrypoint.sh (mkdir -p storages/audiowaveform)